spec/rspec/matchers/built_in/yield_spec.rb

```diff
@@ -67,6 +67,17 @@ def each_arg(*args, &block)
       expect(val).to be_nil
     end
 
+    it 'raises an error if given an invalid count argument' do
+      expect { yield_control.exactly('2') }.to raise_error(ArgumentError)
+      expect { yield_control.at_least(:trice_with_typo) }.to raise_error(ArgumentError)
+      expect { yield_control.at_most(nil) }.to raise_error(ArgumentError)
+    end
+
+    # TODO: This needs implementing.
```

Comment:
Maybe make it a `pending` spec?
```ruby
it 'is yet to support multiple calls to count constraints' do
  pending
  expect { |b| 1.upto(4, &b) }.to yield_control.at_least(:once).at_most(3) }.to raise_error ...
  expect { |b| 1.upto(3, &b) }.to yield_control.at_least(:once).at_most(3) }.not_to raise_error
end
```

Reply:
Ok. I added a test for incompatible constraints (e.g. `include(...).once.twice`) and a pending test for `at_least.at_most`.

Reply:
👍